### ISSUE TITLE
add underline style for DatePicker and Dropdown 

### ISSUE THREAD

**Issue by siminture** [OfficeDev/office-ui-fabric-react#5631]
Add underline style for DatePicker and Dropdown as the TextField component

- [x] Add underline style for DatePicker
- [ ] Add underline style for Dropdown
- [ ] Add underline style for Combox

**Comment by micahgodbolt** [comment]
Can you give a bit more information about this issue/change? Do you have designs? Is this a new component?

**Comment by siminture** [comment]
Plase refer to the TextField component.
I made an example [https://raw.githubusercontent.com/siminture/demos/master/doc/underlined_datepicker.png](https://raw.githubusercontent.com/siminture/demos/master/doc/underlined_datepicker.png)

`<Box>
  <TextField label="Standard TextField" />
  <TextField label="Underlined TextField" underlined />
  <DatePicker label="Standard DatePicker" />
  <DatePicker label="Underlined DatePicker"  />
</Box>`

**Comment by micahgodbolt** [comment]
Yeah, that totally makes sense. All of our pickers are capable of taking a styles object/function, which should allow you to customize them today (which would unblock you if you needed it now). Otherwise, this is a good feature request and we can put it on our backlog.

**Comment by micahgodbolt** [comment]
For reference, we might look at using the textfield for both datepicker, dropdown, combobox so that we don't have to duplicate styles. Or at least abstracted out some basic textfield that each of the 4 components can use

**Comment by ngyukman** [comment]
Hi I was looking for BasePicker with underlined so it could have the same look and feel when using it with underlined TextField. I guess it would make sense to have AutoFill to use TextField underneath

**Comment by micahgodbolt** [comment]
@betrue-final-final does the design language of the textfield line up with pickers etc? They are currently using Autofill, which uses a standard <input> tag. Would be interesting to see what it'd look like replacing that <input> with <TextField> and all of the duplicate functionality that's probably already been written.

@joschect - have you looked into this at all? It'd be great if all the pickers had access to the same <TextField> styles. I know <TextField /> was just converted to css-in-js so that should make it a bit easier to customize.

**Comment by siminture** [comment]
I am looking forward to this revision

**Comment by betrue-final-final** [comment]
We actually have this design pattern in SharePoint already so I would follow that design. @natalieethell, we can discuss in person.

**Comment by natalieethell** [comment]
The underline style has been added for DatePicker (https://github.com/OfficeDev/office-ui-fabric-react/pull/6144) and we'll put the others on the backlog as they're a larger ticket item.

**Comment by alex3683** [comment]
I'm not sure whether to open a new issue or just add it here, so I just do the latter:
The same applies to the PeoplePicker as well. The new O365 OWA client uses this for example.

**Comment by khmakoto** [comment]
Closing in favor of #11345.
